package.json

```diff
@@ -8,6 +8,7 @@
     "build": "webpack --mode=production",
     "watch": "webpack --mode=development --watch",
     "lint": "tslint -p tsconfig.json",
+    "format": "prettier --write \"src/**/*.{ts,js,json,html}\"",
```

Comment:
I :heart: Prettier. Add css,scss,md to the list to auto format these files as well. Works like a charm

Reply:
Thanks.
Yeah, I forgot to add `scss` but I got the vscode prettier extension so the files gets formatted on save. :)